js/poloniex.js

```diff
@@ -386,15 +386,7 @@ module.exports = class poloniex extends Exchange {
         const result = {};
         for (let i = 0; i < marketIds.length; i++) {
             const marketId = marketIds[i];
-            let symbol = undefined;
-            if (marketId in this.markets_by_id) {
-                symbol = this.markets_by_id[marketId]['symbol'];
-            } else {
-                const [ quoteId, baseId ] = marketId.split ('_');
```

Comment:
We can't do `safeSymbol` or `safeMarket` for a reversed pair `[ quoteId, baseId ]` here.